web/public/templates/dojos/user-profile.dust

```diff
@@ -79,7 +79,7 @@
       <div class="panel panel-default" ng-if="!hideProfileBlock('twitter') || !hideProfileBlock('linkedin')">
         <div class="panel-heading">{@i18n key="Social Media"/}</div>
         <div class="panel-body">
-          <a ng-if="profile.twitter && !hideProfileBlock('twitter')" href="https://{{ profile.twitter }}"><i class="fa fa-twitter fa-2x col-lg-3"></i></a>
+          <a ng-if="profile.twitter && !hideProfileBlock('twitter')" href="https://twitter.com/{{ profile.twitter }}"><i class="fa fa-twitter fa-2x col-lg-3"></i>{{ profile.twitter }}</a>
```

Comment:
Cool - I'll look into validating this at the point of input, as we want the twitter handle to be saved in a consistent format, so pretty much as suggested above.

I'll sort within the next 24hrs :metal: